packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -1056,7 +1179,29 @@ class FlameChartNode<T extends FlameChartDataMixin<T>> {
 
   int sectionIndex;
 
-  Widget buildWidget({
+  Paragraph? _paragraph;
+  Paragraph _getParagraph() {
```

Comment:
@dnfield this is basically computed once per node, then laid out based on the size. I have a patch that caches the last layout width and only runs layout if it changes, but no big difference there